default.nix

```diff
@@ -1,53 +1,22 @@
-{ compiler   ? "ghc844"
-, haddock    ? true
-, test       ? true
-, benchmarks ? false
-}:
-with builtins;
+# Builds using system <nixpkgs> and ghc863 with the required package curation.
 let
-  nixpkgs     = import ./nix/nixpkgs.nix {};
-  lib         = nixpkgs.haskell.lib;
-  callCabal2nix = nixpkgs.haskell.packages.${compiler}.callCabal2nix;
-  callPackage = nixpkgs.haskell.packages.${compiler}.callPackage;
+  nixpkgs = import <nixpkgs> {};
```

Comment:
we're not pining `<nixpkgs>` here; it might be good to do that with the version that just works for you now, so that things don't break on a new `nixpkgs` release

Reply:
True. How will this affect cache hits? I'm using the 18.09 channel.

Reply:
In cachix we have a particular version of `nixpkgs`, and if we switch to a new one it's just a matter of `cachix push ouroboros-network ./result` to update the cache.